src/class-wpml-cornerstone-translatable-nodes.php

```diff
@@ -31,17 +45,26 @@ public function get( $node_id, $settings ) {
 							$settings[ $field_key ],
 							$this->get_string_name( $node_id, $field, $settings ),
 							$field['type'],
-							$field['editor_type']
+							$field['editor_type'],
+							$this->get_wrap_tag( $settings )
 						);
 
 						$strings[] = $string;
 					}
 				}
 				if ( isset( $node_data['integration-class'] ) ) {
 					try {
+						/**
+						 * Node object.
+						 *
+						 * @var WPML_Cornerstone_Module_With_Items $node
+						 */
 						$node    = new $node_data['integration-class']();
 						$strings = $node->get( $node_id, $settings, $strings );
+						// phpcs:disable Generic.CodeAnalysis.EmptyStatement.DetectedCatch
 					} catch ( Exception $e ) {
+						// Nothing to do with the exception, we do not handle it.
+						// phpcs:enable
```

Comment:
Asking the same question here just in case I'm right, so to not forget to fix it

Reply:
@kagg-design

Reply:
@andreasciamanna I have this line you asked, before `catch (`. Inside of `catch (` it doesn't work.

Reply:
@kagg-design don't know what you mean, sorry.

Anyway, this is what I did elsewhere:

```php
	// phpcs:disable Generic.CodeAnalysis.EmptyStatement.DetectedCatch
} catch ( Exception $e ) {
	// Nothing to do with the exception, we do not handle it.
}
// phpcs:enable Generic.CodeAnalysis.EmptyStatement.DetectedCatch
```